src/main/scala/com/codacy/client/bitbucket/v2/service/PullRequestServices.scala

```diff
@@ -110,25 +119,38 @@ class PullRequestServices(client: BitbucketClient) {
     postNewComment(author, repo, prId, values)
   }
 
-  def deleteComment(author: String, repo: String, pullRequestId: Int, commentId: Long): RequestResponse[Boolean] = {
-    val url =
-      s"https://bitbucket.org/api/2.0/repositories/$author/$repo/pullrequests/$pullRequestId/comments/$commentId"
+  def deleteComment(
+      author: String,
+      repository: String,
+      pullRequestId: Int,
+      commentId: Long
+  ): RequestResponse[Boolean] = {
+    val pullRequestsUrl = generatePullRequestsUrl(author, repository)
+    val url = s"$pullRequestsUrl/$pullRequestId/comments/$commentId"
 
     client.delete(url)
   }
 
-  def listComments(author: String, repo: String, pullRequestId: Int): RequestResponse[Seq[PullRequestComment]] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$author/$repo/pullrequests/$pullRequestId/comments"
+  def listComments(author: String, repository: String, pullRequestId: Int): RequestResponse[Seq[PullRequestComment]] = {
+    val pullRequestsUrl = generatePullRequestsUrl(author, repository)
+    val url = s"$pullRequestsUrl/$pullRequestId/comments"
 
     client
       .executePaginated(Request(url, classOf[Seq[PullRequestComment]]))
       .map(_.filterNot(_.deleted))
   }
 
   def getPullRequestsReviewers(owner: String, repository: String, prId: Long): RequestResponse[PullRequestReviewers] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$owner/$repository/pullrequests/$prId"
+    val pullRequestsUrl = generatePullRequestsUrl(owner, repository)
+    val url = s"$pullRequestsUrl/$prId"
 
     client.execute(Request(url, classOf[PullRequestReviewers]))
   }
 
+  private def generatePullRequestsUrl(owner: String, repo: String): String = {
```

Comment:
Codacy found an issue: [generatePullRequestsUrl should have a smaller scope by using private[this]](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)